BUILD.md

```diff
@@ -433,9 +433,14 @@ poetry shell
 
 ### If you want to build the installer
 
-* Go to https://dotnet.microsoft.com/download/dotnet-framework and download and install .NET Framework 3.5 SP1 Runtime. I downloaded `dotnetfx35.exe`.
-* Go to https://wixtoolset.org/releases/ and download and install WiX toolset. I downloaded `wix314.exe`.
-* Add `C:\Program Files (x86)\WiX Toolset v3.14\bin` to the path ([instructions](https://web.archive.org/web/20230221104142/https://windowsloop.com/how-to-add-to-windows-path/)).
+Install [.NET SDK](https://dotnet.microsoft.com/en-us/download) version 6 or later.
+Install [WiX toolset](https://wixtoolset.org/) v5 and add the WiX UI extension.
+```
+dotnet tool install --global wix --version 5.0.1
+wix extension add --global WixToolset.UI.wixext/5.0.1
+```
+> [!IMPORTANT]  
+> To prevent compatibility issues, ensure that all installed WiX plugins match at least the major version of the Wix Toolset.
```

Comment:
I had to reopen the Windows shell during the installation of WiX. Let's reflect this in our notes:. Proposed change:

Install [.NET SDK](https://dotnet.microsoft.com/en-us/download) version 6 or later. Then, open a terminal and install [WiX toolset](https://wixtoolset.org/) v5:

```
dotnet tool install --global wix --version 5.*
```

Install the WiX UI extension in a **new terminal**, in order to use the newly installed `wix` tool:

```
wix extension add --global WixToolset.UI.wixext/5.0.1
```

> [!IMPORTANT]  
> To prevent compatibility issues, ensure that all installed WiX plugins match at least the major version of the Wix Toolset.

Reply:
That's odd. I could install both wix and the ui extension without opening a new terminal. Though I think agree we should mention opening the terminal explicitly

Reply:
I'm 99% sure, I also a message that said I needed to open a new terminal to refresh the path. I see a [tutorial](https://learn.microsoft.com/en-us/dotnet/core/tools/global-tools-how-to-use) mentioning this as well:

> If this command fails, you may need to open a new terminal to refresh the PATH.